templates/footer_country_selector.twig

```diff
@@ -0,0 +1,65 @@
+<div class="container">
+	<div class="country-selector-toggle-container">
+		<button
+			class="country-selector-toggle control-toggle"
+			data-bs-toggle="open"
+			data-bs-target="#country-selector"
+			aria-expanded="false"
+			aria-label="{{ data_nav_bar.country_dropdown_toggle }}"
+			data-ga-category="Menu Navigation"
+			data-ga-action="Open Country Selector"
+			data-ga-label="{{ page_category }}">
+				<span class="visually-hidden">{{ __( 'Selected', 'planet4-master-theme' ) }}:</span> {{ website_navbar_title }}
+				<span class="visually-hidden">{{ __( 'Change Country', 'planet4-master-theme' ) }}</span>
+		</button>
+	</div>
+</div>
+<div id="countries-list" class="countries-list">
```

Comment:
Not sure why do you need an `id` here. Correct me if I'm wrong, but I don't see any use with it.